libraries/standard.js

```diff
@@ -1,8 +1,32 @@
-const lib = new Map();
+// Description: Parse a CSV string into an array of objects
 
-// const example_function = function(example_parameter) {
-//     console.log("some example behaviour " + example_parameter);
-// }
-// lib.set("example_function", example_function);
+const csv_parser = function (csv_string) { 
+    // Source: https://stackoverflow.com/questions/8493195/how-can-i-parse-a-csv-string-with-javascript-which-contains-comma-in-data
+    var re_valid = /^\s*(?:'[^'\\]*(?:\\[\S\s][^'\\]*)*'|"[^"\\]*(?:\\[\S\s][^"\\]*)*"|[^,'"\s\\]*(?:\s+[^,'"\s\\]+)*)\s*(?:,\s*(?:'[^'\\]*(?:\\[\S\s][^'\\]*)*'|"[^"\\]*(?:\\[\S\s][^"\\]*)*"|[^,'"\s\\]*(?:\s+[^,'"\s\\]+)*)\s*)*$/;
+    
+    if (!re_valid.test(csv_string)) {
+        console.log("Invalid CSV string ")
+        return null;
+    }
 
-return lib;
```

Comment:
Same thing as removing the map declaration, this should be added back in so we return a map with the CSV function inside of it